resources/nl-core/nl-core-Vaccination-event.xml

```diff
@@ -52,5 +86,41 @@
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole" />
       </type>
     </element>
+    <element id="Immunization.reasonCode">
+      <path value="Immunization.reasonCode" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Immunization.reasonCode:vaccinationIndication">
+      <path value="Immunization.reasonCode" />
+      <sliceName value="vaccinationIndication" />
+      <short value="Why immunization occurred (person-specific)" />

```

Comment:
The `.short`'s of these slices aren't really short. I would propose to align them to the concept names (thus the slice names). In addition, if there's a Dutch equivalent from the dataset, please add them to `.alias`.